src/model/about/iaboutfriend.h

```diff
@@ -10,6 +11,8 @@ class IAboutFriend : public QObject
     Q_OBJECT
 
 public:
+    virtual Friend* getFriend() const = 0;
```

Comment:
Why not `const Friend*`? Looks like it is modified nowhere.

Reply:
Friend* AboutFriend::getFriend() const
{
    return f;
}
Just for compiler report error when "return f". fixed.

Reply:
[This](https://github.com/Diadlo/qTox/commit/f0449346721c6679d0dfd8c547e2d7d56dbd2d2d) is what I meant